src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/YourSingleApprovedApprentice.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Helpers
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.ApprenticeshipViewModel
+
+
+<section class="dashboard-section">
+    <h2 class="section-heading heading-large" id="call-to-action-your-apprentice">
+        Your apprentice
+    </h2>
+
+    <table class="responsive">
+        <tr>
+            <th scope="row" class="tw-35">
+                Name
+            </th>
+            <td class="tw-65" id="apprentice-name">                
+                @(Model?.ApprenticeshipFullName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Training Provider
+            </th>
+            <td id="provider-name">
+                @(Model?.TrainingProvider?.Name)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Course
+            </th>
+            <td id="course-name">
+                @(Model?.CourseName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Start month
+            </th>
+            <td id="course-start-date">
+                @(Model?.CourseStartDate.Value.ToString("MMMM yyyy"))
```

Comment:
Please add CourseStartDateText property to the ApprenticeshipViewModel and set the format in the mapper. Then it's just in one place.